config/jobs/kubernetes-sigs/aws-efs-csi-driver/aws-efs-csi-driver-presubmits.yaml

```diff
@@ -0,0 +1,60 @@
+presubmits:
+  kubernetes-sigs/aws-efs-csi-driver:
+  - name: pull-aws-efs-csi-driver-verify
+    always_run: true
+    decorate: true
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190719-7cdf877-master
+        command:
+        - runner.sh
+        args:
+        - make
+        - verify
+    annotations:
+      testgrid-dashboards: sig-aws-efs-csi-driver
+      testgrid-tab-name: verify
+      description: aws efs csi driver basic code verification
+      testgrid-num-columns-recent: '30'
+  - name: pull-aws-efs-csi-driver-unit
```

Comment:
Could you add a sanity test target here as well since sanity test just got added recently

Reply:
currently it will run under `make test`. Do we want to make it a separate job?

Reply:
ah, saw it. I'm good now